src/components/BookForm.js

```diff
@@ -28,22 +28,31 @@ const BookForm = () => {
   };
 
   return (
-    <div>
-      <h1>ADD NEW BOOK</h1>
-      <form>
+    <div className="container-form">
+      <h1 className="title-form">ADD NEW BOOK</h1>
+      <form className="form">
         <input
+          className="input-title-book"
           type="text"
           placeholder="Book Title"
           value={title}
           onChange={(e) => setTitle(e.target.value)}
         />
-        <input
-          type="text"
-          placeholder="Author"
-          value={author}
-          onChange={(e) => setAuthor(e.target.value)}
-        />
-        <Button text="Add Book" handleAddBook={handleAddBook} />
+        <div className="input-container">
+          <div className="input-author-book-container">
+            <input
+              className="input-author-book"
+              type="text"
+              placeholder="Category"
+              value={author}
+              onChange={(e) => setAuthor(e.target.value)}
+            />
```

Comment:
- [ ] Nice work on the style I can see it's wonderful, how ever I think the category of the book should be taken form a drompdown list or `<select><select>` tag rather that a simple input

Reply:
@andersonlebon Hi! the dropdown functionality is not a requirement, as indicated in the project guides, just recreate the arrow inside the input as in the layout, if you read the requirements it says to add new elements to the project but without its functionality: 

> 1. Add any missing JSX elements to your project (without adding extra functionality)
> 2. Your final design should match Zeplin's design (this includes all elements + the same styling)
> 3. You may use the CSS rules detailed in Zeplin to style the website.

the dropdown should not work as such, it should just be there.

Reply:
Hey @rodrigopenayo98, eventhough the requirement does not show directly that the dropdown should be there, the design guidelines show it(`the design should match the Zeplin's design including all elements ....), therefore at a place of an select input we should not use a text input.